packages/api/src/models/expenseTypeModel.js

```diff
@@ -24,6 +24,10 @@ class ExpenseType extends baseModel {
     return 'expense_type_id';
   }
 
+  static get hidden() {
```

Comment:
Thank you for explaining what this does re:deleted! I'll add it to revenueTypes too!